multiples.js

```diff
@@ -5,8 +5,13 @@
  */
 exports.sumOfAMultiple = function( n ) {
   var sum = 0;
-
-  // do your work here
-
+  var arr = [];
```

Comment:
This isn't used anywhere. You should get rid of variables that aren't being used.